integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -4083,6 +4091,17 @@ msgstr ""
 "Chat-Tickets automatisch abonnieren sollen. Beachten Sie, dass diese "
 "Accounts volle Gruppenberechtigungen für folgende Gruppe haben müssen:"
 
+#: cms/models/regions/region.py
+msgid "Chat beta tester percentage"
+msgstr "Chat Beta-Tester Prozentsatz"
+
+#: cms/models/regions/region.py
+msgid ""
+"Percentage of users selected as beta testers for the Integreat Chat feature"
+msgstr ""
+"Prozentsatz der Nutzer, die als Beta-Tester für den Integreat-Chat "
+"ausgewählt werden sollen"
+
```

Comment:
This line in the translation file `Ein Zugangs-Token wurde für diese Region existiert, wird hier jedoch aus Sicherheitsgründen nicht angzeigt.` has a typo. I think we can just delete the "wurde". Maybe it makes sense to also correct it in this PR? :)

Reply:
Yes, good catch, thank you!